com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/ResourceSetOptions.java

```diff
@@ -52,4 +53,12 @@ public static void setInstallDerivedState(final @NonNull ResourceSet resourceSet
     resourceSet.getLoadOptions().put(INSTALL_DERIVED_STATE, installDerivedState);
   }
 
+  public static boolean skipAST(final @NonNull ResourceSet resourceSet) {
+    Object object = resourceSet.getLoadOptions().get(SKIP_AST);
+    return object != null && (boolean) object; // default is false
+  }
+
+  public static void setSkipAST(final @NonNull ResourceSet resourceSet, final @Nullable Boolean installDerivedState) {
```

Comment:
the name `installDerivedState` needs to be changed, right?

Reply:
I don't understand why it needs to be changed, please explain.

Reply:
the second parameter `installDerivedState` should be named `installAST`, shouldn't it? Or with the naming I suggest, `installModel` and the method called `setSkipModel`

Reply:
Yes it should. I have done the rename you suggested and used "model" instead of "AST".